roxie/udplib/udpsha.cpp

```diff
@@ -516,10 +516,11 @@ void sanityCheckUdpSettings(unsigned receiveQueueSize, unsigned numSenders, __ui
         trace("udpRequestTimeout", udpRequestTimeout, (2 * minLatencyNs + minTimeForPermitPackets) * 2 / 5, 10);
         trace("udpResendDelay", udpResendDelay, minTimeForAllPackets, 10);
         DBGLOG("udpMaxPendingPermits: %u [%u..%u]", udpMaxPendingPermits, udpMaxPendingPermits, udpMaxPendingPermits);
-        DBGLOG("udpMaxClientPercent: %u [%u..%u]", udpMaxClientPercent, 100, 500);
+        DBGLOG("udpMaxClientPercent: %u [%u..%u]", udpMaxClientPercent, 100, 800);
```

Comment:
800 or 600 ?
Above is:
```
unsigned udpMaxClientPercent = 600;
```

Reply:
Is 800 just a suggestion for a reasonable max ?

Reply:
It should probably be udpMaxPendingPermits * 100 i.e. use all of them.  I'll change.